app/courses/page.tsx

```diff
@@ -24,24 +28,49 @@ const Paths = async () => {
   // return <pre>{JSON.stringify(paths, null, 2)}</pre>;
 
   return (
-    <Flex flexDir={"column"} gap={4}>
-    <Banner/>
-      <Heading fontSize={"lg"}>Learning paths</Heading>
-      <SimpleGrid columns={{ sm: 1, lg: 2 }} gap={4}>
-        {paths.data.map((path, key) => (
-          <Path
-            key={key}
-            isFree={path.attributes.isFree}
-            slug={path.attributes.title.replaceAll(" ", "-").toLowerCase()}
-            level={path.attributes.level}
-            title={path.attributes.title}
-            description={path.attributes.description}
-            image={platypusLogo}
-            publishedAt={new Date().toDateString()}
-            articles={path.attributes.articles}
-          />
-        ))}
-      </SimpleGrid>
+    <Flex flexDir={"column"} gap={12}>
```

Comment:
Was this change not also in the last PR? Or am I going crazy?

Reply:
I think a bunch will be as I just made a new branch after committing?

Reply:
Yeah I think you might have baked the git ops here. Hopefully github is smart enough to not see these as conflicts.

Reply:
Ok so best to do one merge at a time? I just didnt want to pull main and lose the changes I had made as was using some as a guide